custom-words.txt

```diff
@@ -131,6 +131,7 @@ artifacturi
 asciifolding
 Asns
 aspnet
+ASSDZRS
```

Comment:
why is this needed?

Reply:
SDK validation threw error for the new SSD ZRS test added, so the options were to change the test name or add the custom word.

Reply:
ASSDZRS isnt custom word.. either add it as SSDZRS or change the test name to CreateAManagedDiskWithSSDZRSAccountType.json